backend/src/common/util/execution_spec_test.go

```diff
@@ -135,7 +135,7 @@ func TestExecutionSpec_NewExecutionSpecFromInterface(t *testing.T) {
 	execSpec, err = NewExecutionSpecFromInterface(TektonPipelineRun, test)
 	assert.Empty(t, execSpec)
 	assert.Error(t, err)
-	assert.EqualError(t, err, "InternalServerError: type:PipelineRun: ExecutionType is not supported")
+	assert.EqualError(t, err, "Invalid input error: not PipelineRun struct")
```

Comment:
nit: there should be a better error message

Reply:
It follows the same message for Argo workflow: https://github.com/kubeflow/pipelines/blob/master/backend/src/common/util/workflow.go#L94